tests/all_good_test.py

```diff
@@ -217,11 +217,11 @@ class TestAllGood(object):
         ("[a + b]", _Add(a, b)),
         ("\\frac{d}{dx} [ \\tan x ]", Derivative(tan(x), x)),
         ("2\\overline{x}", 2 * Symbol('xbar', real=True)),
-        ("2\\overline{x}_n", 2 * Symbol('xbar_{n}', real=True)),
-        ("\\frac{x}{\\overline{x}_n}", x / Symbol('xbar_{n}', real=True)),
-        ("\\frac{\\sin(x)}{\\overline{x}_n}", sin(Symbol('x', real=True)) / Symbol('xbar_{n}', real=True)),
+        ("2\\overline{x}_n", 2 * Symbol('xbar_n', real=True)),
```

Comment:
@builtbywill should one of these have a subscript that is more than one char long?

Reply:
{x}_{i,j} for example

Reply:
@lily514 nice catch, yeah let me add some more tests